package.json

```diff
@@ -62,11 +62,12 @@
     "benchmark": "^2.1.4",
     "bole": "^3.0.2",
     "bunyan": "^1.8.12",
+    "cross-env": "^5.2.0",
     "docsify-cli": "^4.2.1",
     "execa": "^1.0.0",
     "fastbench": "^1.0.1",
     "flush-write-stream": "^2.0.0",
-    "fresh-require": "^1.0.3",
```

Comment:
This dependency generates security warning noise during audit, and is entirely unnecessary - there is an up-to-date alternative from well-respected sindresorhus.